src/mbgl/map/map_context.cpp

```diff
@@ -32,7 +32,10 @@ MapContext::MapContext(View& view_, FileSource& fileSource, MapData& data_)
       data(data_),
       asyncUpdate(std::make_unique<uv::async>(util::RunLoop::getLoop(), [this] { update(); })),
       asyncInvalidate(std::make_unique<uv::async>(util::RunLoop::getLoop(), [&view_] { view_.invalidate(); })),
-      texturePool(std::make_unique<TexturePool>()) {
+      texturePool(std::make_unique<TexturePool>()),
+      savedPixelRatio(data.pixelRatio),
+      annotationSpriteStore(nullptr),
```

Comment:
`nullptr` is the default value for `std::unique_ptr`, so it's not necessary to initialize `annotationSpriteStore` and `annotationSpriteStore` explicitly.